packages/core/src/providers/webex.ts

```diff
@@ -0,0 +1,100 @@
+/**
+ * <div style={{backgroundColor: "#000", display: "flex", justifyContent: "space-between", color: "#fff", padding: 16}}>
+ * <span>Built-in <b>Webex</b> integration.</span>
+ * <a href="https://webex.com">
+ *   <img style={{display: "block"}} src="https://authjs.dev/img/providers/webex.svg" height="48" width="48"/>
+ * </a>
+ * </div>
+ *
+ * @module providers/webex
+ */
+import type { OAuthConfig, OAuthUserConfig } from "./index.js"
+
+/**
+ * The returned user profile from Webex when using the profile callback.
+ *
+ * Please refer to {@link https://developer.webex.com/docs/api/v1/people/get-my-own-details|People - Get My Own Details}
+ * on Webex Developer portal for additional fields. Returned fields may vary depending on the user's role, the OAuth
+ * integration's scope, and the organization the OAuth integration belongs to.
+ */
+export interface WebexProfile extends Record<string, any> {
+  id: string;
+  emails: [string, ...string[]];
```

Comment:
Is this supposed to be a string or an array of strings? i.e. `string | string[]`?

Reply:
The later one. It's an array of string, with at least one element.

Reply:
I have updated it to `string[]`.